src/main/io/rc_controls.c

```diff
@@ -392,260 +400,126 @@ void applyStepAdjustment(controlRateConfig_t *controlRateConfig, uint8_t adjustm
     }
     switch(adjustmentFunction) {
```

Comment:
I spent some thoughts about making a table but that would require collecting all the pointers at startup and there will be some confusion about the adjustments which change two values at time, so there would also offsets be needed to find the right row.